assets/css/main.css

```diff
@@ -4245,3 +4245,12 @@ a.comms_guide_res_table strong {
   max-width: 40%;
   margin-bottom: 1em;
 }
+
+#BackToTop {
+  bottom: 10vh;
+  right: 10vw;
+  position: fixed;
+  z-index: 999;
+  border: none;
+  border-radius: 50%;
+}
```

Comment:
Adding padding fixes the oval shape, i.e., `padding: 2.5em 2.5em;`, but this causes an issue with the arrow positioning which should then be fixed.

Also the size should be decreased a little bit, i.e., `font-size: 9px;` with the above mentioned padding would look good size wise.

![image](https://user-images.githubusercontent.com/21331903/81398409-d5866080-9131-11ea-9ff2-ff6eee00358b.png)

Reply:
I have done requested changes.
Now the button looks like:
![button](https://user-images.githubusercontent.com/63332774/81419757-5186a580-913e-11ea-8e30-70d62f07d968.png)
Is this fine?